mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,45 @@
+import {ReactElement} from "react";
+
+/**
+ * Params for {@link Text}
+ */
+interface TextProps {
+
+  /**
+   * Displayed values
+   */
+  value: string | number;
+
+  /**
+   * Displayed placeholder
+   */
+  placeholder: string;
```

Comment:
Text shouldn't have prop placeholder. Text can have only value

Reply:
Deleted the placeholder for the Text component